yo1k/tic_tac_toe/test/core/test_game.py

```diff
@@ -112,10 +112,10 @@ def test_win_condition(self):
                  Cell(0, 2), False)]
         for cells, cell, expect in args_and_expect_list:
```

Comment:
`Mypy` does not inference the right type for `cells` and the syntax [does not allow one to annotate the types of variables when tuple unpacking is used](https://www.python.org/dev/peps/pep-0526/#:~:text=Note%20that%2C%20although%20the%20syntax%20does%20allow%20tuple%20packing%2C%20it%20does%20not%20allow%20one%20to%20annotate%20the%20types%20of%20variables%20when%20tuple%20unpacking%20is%20used%3A).